### ISSUE TITLE
CodePipeline: Approval process to be driven from backstage interface.

### ISSUE THREAD

**Issue by anshulgargamway** [awslabs/aws-codeservices-plugins-for-backstage#39]
Currently for approving "Manual Approval" stage in pipeline we need to do manually through AWS Console. Is it possible to modify the widget in a way that we can directly approve from backstage ofcourse user with required permission should be able to do that.

**Comment by niallthomson** [comment]
Thanks for raising this feature request!

It looks like this would be dependent on #38 since approvals are at the action level.

**Comment by niallthomson** [comment]
We have started a new project to better scale our efforts regarding Backstage plugins. The functionality in this repository is being migrated. If this issue is still relevant please create a new issue in this repository:

https://github.com/awslabs/backstage-plugins-for-aws
